sidebars.js

```diff
@@ -8,154 +8,180 @@
 module.exports = {
   "docs": [
     {
-      "Welcome": [
+      "Get started": [
       "overview",
-      "saas-vs-self-hosted",
+      "self-hosted/install/preparation",
       {
         "type": "category",
-        "label": "Getting Started",
+        "label": "Installing Okteto",
         "items": [
-          "getting-started",
-          "using-dev-envs"
+          "self-hosted/install/deployment",
+          {
+            "type": "category",
+            "label": "Cloud provider guides",
+            "items": [
+              "self-hosted/eks",
+              "self-hosted/aks",
+              "self-hosted/civo",
+              "self-hosted/do",
+              "self-hosted/gke"
+            ]
+          },
         ]
-      }
+      },
+      "using-dev-envs"
```

Comment:
I think we are missing this one, which explains how to install and configure the Okteto CLI:
https://www.okteto.com/docs/getting-started/
This way, the journey is:
- Install SH
- Install CLI and configure it
- Deploy your first environment (using our sample app)
-